spec/features/gardens/show_spec.rb

```diff
@@ -184,6 +184,9 @@
       sensors = File.read('spec/fixtures/sensors.json')
       stub_request(:get, "#{ENV['BE_URL']}/api/v1/gardens/3/sensors").to_return(status: 200, body: sensors)
 
+      sensor = File.read('spec/fixtures/sensor.json')
+      stub_request(:get, "https://solar-garden-be.herokuapp.com/api/v1/sensors/#{@sensor1[:id]}").to_return(status: 200, body: sensor)
```

Comment:
'https://solar-garden-be.herokuapp.com' should be replaced with "#{ENV['BE_URL']}".